src/components/MoviePage.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import logo from '../logo.svg';
+import '../App.css';
+
+class MoviePage extends Component {
+  render() {
+    return (
+      <div>

```

Comment:
You can use `React.Fragment` or shortcut for this `<>`

Reply:
Done